.github/workflows/python.yml

```diff
@@ -303,19 +316,13 @@ jobs:
               with:
                   submodules: recursive
 
-            - name: Setup Python for self-hosted Ubuntu runners
-              run: |
-                  sudo apt update -y
-                  sudo apt upgrade -y
-                  sudo apt install python3 python3-venv python3-pip -y
-
             - name: Build Python wrapper
               uses: ./.github/workflows/build-python-wrapper
               with:
-                  os: ${{ matrix.host.OS }}
-                  target: ${{ matrix.host.TARGET }}
+                  os: ubuntu
+                  target: aarch64-unknown-linux-gnu
                   github-token: ${{ secrets.GITHUB_TOKEN }}
-                  engine-version: ${{ matrix.engine.version }}
+                  engine-version: 7.2
```

Comment:
Skip this - the task shouldn't install valkey if no version given

Reply:
So CD needs to install just when arrive to release testing

Reply:
unless you're testing server modules - workflow uses a pre-installed meme DB there